src/Rule/AtLeast.php

```diff
@@ -15,7 +15,9 @@
 use Yiisoft\Validator\WhenInterface;
 
 /**
- * Checks if at least {@see AtLeast::$min} of specified attributes are filled.
+ * Defines validation options to check that a minimum number of specified attributes are filled.
```

Comment:
Add info about supported values: array and object with public properties.

Reply:
Agree. It also could be added in the corresponding property too.